__tests__/routes/profiles.test.js

```diff
@@ -153,8 +153,8 @@ describe('Profile Router', () => {
         expect(authRequired).toBeCalled();
       });
 
-      it('responds with status 200', async () => {
-        const expected = 200;
+      it('responds with status 201', async () => {
+        const expected = 201;
```

Comment:
Great catch! Nice job correcting both the printed console and actual coded res numbers!